tasks/setup-RedHat.yml

```diff
@@ -6,8 +6,10 @@
     enablerepo: "{{ php_enablerepo | default(omit, true) }}"
   notify: restart webserver
 
-- name: Enable relevant repos so we get upgrades
+- name: Enable PHP repo so we get upgrades
   yum_repository:
     name: "{{ item }}"
+    baseurl: "{{ remi_repo_url }}"
+    gpgkey: "{{ remi_repo_gpg_key_url }}"
     enabled: yes
-  with_items: "{{ php_enablerepo.split(',') }}"
+  with_items: "{{ php_eenablerepo.split(',') }}"
```

Comment:
Can you omit `baseurl` and `gpgkey` here as they'll vary when iterating over `php_eenablerepo.split(',') ` and should already be defined in `/etc/yum.repos.d`?

Reply:
I think, basically, no:

```
TASK [willfurnass.php : Enable PHP repo so we get upgrades] ********************
failed: [vagrant-web1] (item=remi-php73) => {"ansible_loop_var": "item", "changed": false, "item": "remi-php73", "msg": "Parameter 'baseurl', 'metalink' or 'mirrorlist' is required."}
```